### PR TITLE
add steps on how to point to your local content folder

### PR DESCRIPTION
**Pull request by aronasorman**
Seems to be a frequent question that deserves its own entry into our frequently asked questions section.

### CODE REVIEW

File: sphinx-docs/faq.rst [hunk 1]
```diff
@@ -18,6 +18,19 @@ Please follow the instructions on our `Github Wiki`_ for reporting bugs.
...
+-----------------------------------------
+
+If you want to change your installation's content folder from the default (say, to point to a shared folder across installations), here are the steps you need to do:
+
+#. If it doesn't already exist, create a file named local_settings.py in the ka-lite/kalite folder (the one containing settings.py)
+#. Add the line ``CONTENT_ROOT="[full path to your videos directory]"``, making SURE to include an OS-specific slash at the end (see examples) and encapsulate it in quotes.
+    **For example, on Windows:** ``CONTENT_ROOT="C:\\videos_location\\"``
+
+    **For example, on Linux:** ``CONTENT_ROOT="/home/me/videos_location/"``
+#. Restart your server. If you are unsure on how to do this, please see `Restarting Your Server`_ .
```
Comment (MCGallaspy) [inline sphinx-docs/faq.rst]: This probably doesn't do what you think it does. If you want to cross reference an internal location, [check this out](http://sphinx-doc.org/markup/inline.html#cross-referencing-arbitrary-locations).

File: sphinx-docs/faq.rst [hunk 2]
```diff
@@ -18,6 +18,19 @@ Please follow the instructions on our `Github Wiki`_ for reporting bugs.
...
+How do I change KA Lite's content folder?
+-----------------------------------------
+
+If you want to change your installation's content folder from the default (say, to point to a shared folder across installations), here are the steps you need to do:
+
+#. If it doesn't already exist, create a file named local_settings.py in the ka-lite/kalite folder (the one containing settings.py)
+#. Add the line ``CONTENT_ROOT="[full path to your videos directory]"``, making SURE to include an OS-specific slash at the end (see examples) and encapsulate it in quotes.
+    **For example, on Windows:** ``CONTENT_ROOT="C:\\videos_location\\"``
+
+    **For example, on Linux:** ``CONTENT_ROOT="/home/me/videos_location/"``
```
Comment (MCGallaspy) [inline sphinx-docs/faq.rst]: I would remove the indentation. It usually has special meaning in rst.

Comment (aronasorman) [inline sphinx-docs/faq.rst]: Without indentation, I'm getting this:

image

I think having the indentation is more readable. Unless there's some other thing that I need to do to make it look good. Recommendations?

Comment (aronasorman) [inline sphinx-docs/faq.rst]: For comparison, this is how it looks with indentation:

image

**Comment by MCGallaspy** [comment]
Just docs, looks safe to merge. Will run the docs-building command, too.

**Pull request merged**